### PR TITLE
get rid of hardcoded tomcat version from Dockerfiles

### PR DESCRIPTION
**Pull request by AtsushiNemoto**
Get rid of "tomcat-9.0.17" from Dockerfiles.
This reqeest includes pull request #66 (fix sw360populated/Dockerfile).

### CODE REVIEW

File: docker-images/sw360empty/Dockerfile [hunk 1]
```diff
@@ -21,4 +21,4 @@ COPY tomcatdeploy.sh /usr/local/bin/tomcatdeploy.sh
 COPY docker-entrypoint.sh /usr/local/bin/entrypoint.sh
 
 ENTRYPOINT ["/usr/local/bin/entrypoint.sh"]
-CMD ["tail", "-f", "/opt/sw360/tomcat-9.0.17/logs/catalina.out"]
```
Comment (maxhbr) [inline docker-images/sw360empty/Dockerfile:24]: does this get the following env variable?

https://github.com/sw360/sw360chores/blob/35f7dbeb49a1fb012db0df21219f6f9ec5c2ef06/docker-images/sw360empty/docker-entrypoint.sh#L54

**Comment by maxhbr** [comment]
This contains #66 but drops all its changes? better rebase to not contain #66.

File: docker-images/sw360populated/Dockerfile [hunk 2]
```diff
@@ -14,7 +14,8 @@ COPY _deploy/liferay/* /opt/sw360/deploy/liferay/
 RUN set -ex \
  && for war in /opt/sw360/deploy/tomcat/*.war; do \
       folder=$(basename $war .war); \
-      mkdir -p /opt/sw360/tomcat-*/webapps/$folder; \
-      unzip -q $war -d /opt/sw360/tomcat-*/webapps/$folder; \
+      (cd /opt/sw360/tomcat-*; \
```
Comment (maxhbr) [inline docker-images/sw360populated/Dockerfile:17]: This is an improvement in code quality, but should not change any functionality or logic, right?

File: docker-images/sw360empty/Dockerfile (hunk 1 above)
Comment (AtsushiNemoto) [reply docker-images/sw360empty/Dockerfile:24]: Yes.

**Comment by AtsushiNemoto** [comment]
> This contains #66 but drops all its changes? better rebase to not contain #66.

Yes, but #66 is a fix and this is a cleanup.  I would like to keep these changes separated.

File: docker-images/sw360populated/Dockerfile (hunk 2 above)
Comment (AtsushiNemoto) [reply docker-images/sw360populated/Dockerfile:17]: Yes.

**Review by maxhbr — APPROVED** [review]

**Pull request merged**